src/module/Ui/templates/layout/navigation/partial/breadcrumb.phtml

```diff
@@ -0,0 +1,21 @@
+<ol class="breadcrumb">
+<?php
+    foreach ($this->pages as $i => $page) {
+        $html = [];
+        $html[] = '<li>';
+
+        if ($i + 1 < count($this->pages)) {
```

Comment:
`if ($i + 1 < $pagesCount) {`

and add

`$pagesCount = count($this->pages)`

before foreach